app/views/places/show.html.haml

```diff
@@ -32,4 +32,28 @@
       %td= @place.rate.to_s
 
 %p
-  = link_to "<-- Back", root_path
\ No newline at end of file
+  = link_to "<-- Back", root_path
+
+
+%h4 Add new review for  this place
+
+.content
+  = render partial: 'reviews/reviews', place: @place
+
+
+
+  %table.table.table-hover
```

Comment:
Cade related to reviews can be moved to reviews partial (like the form for reviews)